Features/PrefixCommand.cs

```diff
@@ -0,0 +1,36 @@
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+using DSharpPlus.Commands.ContextChecks;
+using ArtcordAdminBot.Features.Helpers;
+
+
+namespace ArtcordAdminBot.Features
+{
+    public class PrefixCommand
+    {
+        private readonly IDatabaseService _databaseService;
+
+        public PrefixCommand(IDatabaseService databaseService)
+        {
+            _databaseService = databaseService;
```

Comment:
Use using statements, see the other comment I made on the other file for info

Reply:
if the database service is using the "using" statement then do i need to use using statements here?

Reply:
idk what im doing >_<

Reply:
Whenever you access the database, you should use them. If you use them in the database service then you're not gonna need it when you access the database service